### PR TITLE
OCPBUGS#23026: fix rhel attribute in repos

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OCPBUGS-23026

Link to docs preview:
[rhel-9; install from RPMs](https://67618--docspreview.netlify.app/microshift/latest/microshift_install/microshift-install-rpm#installing-microshift-from-rpm-package_microshift-install-rpm)
[rhel9 in Image Builder URL and command block](https://67618--docspreview.netlify.app/microshift/latest/microshift_install/microshift-embed-in-rpm-ostree#adding-microshift-repos-image-builder_microshift-embed-in-rpm-ostree)
[second command block; example blueprint](https://67618--docspreview.netlify.app/microshift/latest/microshift_install/microshift-embed-in-rpm-ostree#adding-microshift-service-to-blueprint_microshift-embed-in-rpm-ostree)
[deploy apps code blocks](https://67618--docspreview.netlify.app/microshift/latest/microshift_running_apps/microshift-applications#microshift-manifests-overview_applications-microshift)

QE review:
- [x] QE has approved this change.

 
SME approved

### CODE REVIEW

File: _attributes/attributes-microshift.adoc [hunk 1]
```diff
@@ -10,7 +10,7 @@
 :microshift-short: MicroShift
 :product-registry: OpenShift image registry
 :product-version: 4.14
-:rhel-major: rhel-9
+:rhel-major: rhel9
```
[the remark this answers is not in the document]
Comment (dhensel-rh) [reply _attributes/attributes-microshift.adoc:13]: @ShaunaDiaz Is the attribute `rhel9` used to create the line in Procedure 1. ?

`https://67618--docspreview.netlify.app/microshift/latest/microshift_install/microshift-install-rpm#installing-microshift-from-rpm-package_microshift-install-rpm`

[the remark this answers is not in the document]
Comment (ShaunaDiaz) [reply _attributes/attributes-microshift.adoc:13]: @dhensel-rh Only the `rhel9` portion, for example:
url = "https://cdn.redhat.com/content/dist/layered/{rhel-major}/$(uname -m)/fast-datapath/os"

**Comment by jab-rh** [comment]
@ShaunaDiaz, 4.15 as well? Thanks!

**Comment by ShaunaDiaz** [comment]
@jab-rh Oh yes! Thanks for asking

**Pull request merged**